src/test/ui/cast/cast-ptr-to-int-const.stderr

```diff
@@ -1,17 +1,19 @@
-error[E0658]: casting pointers to integers in constants is unstable (see issue #51910)
+error[E0658]: casting pointers to integers in constants is unstable
   --> $DIR/cast-ptr-to-int-const.rs:5:9
    |
 LL |         main as u32
    |         ^^^^^^^^^^^
    |
+   = note: for more information, see tracking issue #51910
```

Comment:
Since you are pulling this out into its own note, I think you could reasonably post the full link:

```suggestion
   = note: for more information, see https://github.com/rust-lang/rust/issues/51910
```